pkg/kubelet/kubelet_pods.go

```diff
@@ -1156,7 +1157,7 @@ func (kl *Kubelet) RunInContainer(podFullName string, podUID types.UID, containe
 
 	var buffer bytes.Buffer
 	output := ioutils.WriteCloserWrapper(&buffer)
-	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil)
+	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil, 0)
```

Comment:
> Do you know what RunInContainer is used for?

Not really, at least not "historically speaking". It seems it is only used in https://github.com/kubernetes/kubernetes/blob/e4b41dd2920eb98402d745931964f11279a9ff2e/pkg/kubelet/server/server.go#L624 apart from tests. That can probably be refactored to eliminate the `RunInContainer` method.